lib/newrelic-fluentd-output/version.rb

```diff
@@ -1,3 +1,3 @@
 module NewrelicFluentdOutput
-  VERSION = "1.1.11"
+  VERSION = "1.1.12"
```

Comment:
Please update the `minor` version instead
```suggestion
  VERSION = "1.2.0"
```